File: C99/3342.c
#include <stdio.h>
#include <math.h>

int main() {
    int i,m,b,p;
    scanf("%i",&i);
    m=pow(i,2);
    b = m%2 == 0 ? m / 2 : ceil(m / 2)+1;
    p = m%2 == 0 ? m / 2 : floor(m / 2);
    printf("%i casas brancas e %i casas pretas\n",b,p);
    return 0;
}

File: C#/3342.cs
using System;

class URI {

    static void Main(string[] args) {

        int i = Int32.Parse(Console.ReadLine());
        int m = 0;
        double aux, b, p;

        m = i * i;
        aux = m / 2.0;
        b = m % 2 == 0 ? aux : Math.Ceiling(aux);
        p = m % 2 == 0 ? aux : Math.Floor(aux);

        Console.WriteLine(b + " casas brancas e " + p + " casas pretas");
    }
}